frontend/projects/upgrade/src/app/core/local-storage/local-storage.service.ts

```diff
@@ -66,7 +93,8 @@ export class LocalStorageService {
     };
 
     const state = {
-      experiments: experimentState, // experiment state,
+      experiments: experimentState,
+      featureFlagState: featureFlagState,
```

Comment:
@zackcl Change this to `featureFlags` for local storage to work correctly

Reply:
That worked, thank you! I created a PR that fixes this: https://github.com/CarnegieLearningWeb/UpGrade/pull/2005